src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: "Avenir Book";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+.search-bar {
+  display: flex;
+  align-items: center;
+  margin-top: 20px;
+}
+
+.search-bar--large {
+  height: 70px;
+  padding-left: 26px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search-bar--large:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar--large:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-bar--small {
+  height: 42px;
+  padding-left: 13px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
```

Comment:
The code for small and big search bars is duplicated.
It would be better to set styles for search-bar class and override for small one only those you need. for instance:

```css
.search-bar {
  height: 70px;
  padding-left: 26px;
  border-radius: 4px;
  border: 1px solid #e1e7ed;
  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
}

.search-bar--small {
  height: 42px;
  padding-left: 13px;
}
``` 

with this approach you write less code and don't duplicate it)

Reply:
fix everywhere